sound/soc/sof/mediatek/mt8195/mt8195-loader.c

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// Copyright (c) 2021 Mediatek Corporation. All rights reserved.
+//
+// Author: YC Hung <yc.hung@mediatek.com>
+//
+// Hardware interface for mt8195 DSP code loader
+
+#include <sound/sof.h>
+#include "mt8195.h"
+#include "../../ops.h"
+
+void hifixdsp_boot_sequence(void *pdev, u32 boot_addr)
```

Comment:
Why `void *`?

Reply:
I declare this function in mt8195.h and it doesn't include "snd_sof_dev.h" so I use void * for it.  Should I modify it? thanks

Reply:
Just forward declare the sdev struct in the header, no need to include anything else:
```c
struct snd_sof_dev;
```